src/cmd.rs

```diff
@@ -566,6 +566,75 @@ pub fn cluster_health(cluster_handle: &Rados) -> RadosResult<ClusterHealth> {
     Ok(serde_json::from_str(&return_data)?)
 }
 
+/// Check with the monitor whether a given key exists
+pub fn get_config_exists(cluster_handle: &Rados, key: &str) -> RadosResult<bool> {
+    let cmd = json!({
+        "prefix": "config-key exists",
+        "key": key,
+        "format": "json"
+    });
+
+    let result = cluster_handle.ceph_mon_command_without_data(&cmd)?;
+    let return_data = String::from_utf8(result)?;
+    let mut l = return_data.lines();
+    match l.next() {
+        Some(val) => Ok(bool::from_str(val)?),
+        None => Err(RadosError::Error(format!(
+            "Unable to parse config-key exists output: {:?}",
+            return_data,
+        ))),
+    }
+}
+
+/// Ask the monitor for the value of the configuration key
+pub fn get_config_key(cluster_handle: &Rados, key: &str) -> RadosResult<String> {
+    let cmd = json!({
+        "prefix": "config-key",
+        "key": key,
+        "format": "json"
+    });
+
+    let result = cluster_handle.ceph_mon_command_without_data(&cmd)?;
+    let return_data = String::from_utf8(result)?;
+    let mut l = return_data.lines();
+    match l.next() {
+        Some(val) => Ok(val.to_string()),
+        None => Err(RadosError::Error(format!(
+            "Unable to parse config-key get output: {:?}",
+            return_data,
+        ))),
+    }
+}
+
+/// Remove a given configuration key from the monitor cluster
+pub fn get_config_remove(cluster_handle: &Rados, key: &str, simulate: bool) -> RadosResult<()> {
```

Comment:
`pub fn remove_config` maybe, instead of `get_config_remove`?

Reply:
Good idea!  I'll rename these